rtos/rtx2/mbed_rtx_conf.h

```diff
@@ -32,7 +32,10 @@
 #endif
 #endif
 
-#define OS_DYNAMIC_MEM_SIZE         0
+/* mbed OS uses this macro as an on/off switch for RTX dynamic memory. The value has to be either 0 to disable
+   or any other number (dividable by 8) to enable.
+ */
+#define OS_DYNAMIC_MEM_SIZE         8
```

Comment:
So this makes RTX set up an 8-byte heap, and pass it to your osRtxMemoryInit? But your osRtxMemoryAlloc actually uses the real heap.

Seems a reasonable trick, even if it does end up having 8 bytes of unused RAM. (Does it?)